l10n_vn_account_invoice_number/README.rst

```diff
@@ -0,0 +1,85 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+======================
+Account Invoice Number
+======================
+
+## Expected behavior:
```

Comment:
This is not a proper RST sytax

Also, the description is not very clear. I would recommend:

```rst
Vietnamese regulations (from tax authorities) requires to display 3 values on the reports of accounting transactions:

The 3 values to display are:

- Tax Invoice Number
- Tax Invoice Form
- Tax Invoice Series: The references of the series of invoice (the reference of the accounting booklet)

In Odoo, the following types of records need to store those values:

- Customer Invoice
- Supplier Invoice
- Sale Receipt (type is Pay Later)
- Purchase Receipt (type is Pay Later)

In Odoo, we need to store this information whenever the issuer of the accounting document is in Vietnam, so this module adds a control when validating a customer/supplier invoice for a issuer based in Vietnam.

```